cli/mender-artifact/main.go

```diff
@@ -219,6 +219,10 @@ func getCliContext() *cli.App {
 				"creating artifact from snapshot (i.e. FILE " +
 				"contains 'ssh://' schema)",
 		},
+		cli.BoolFlag{
+			Name: "quiet, q",
```

Comment:
Since this only affects the progressbar, I think it should be called `--no-progress` and I don't think it needs a short hand form. The client is still very chatty in other respects, and `--quiet` is something we might want to implement later.

Reply:
Fair point. I agree.

Reply:
And, 
> The client is still very chatty [...]
`mender-artifact` you mean? It's not very chatty.

Reply:
Hmm, also a fair point. But if we really want that, then I think we should set log level to Error level as well, so that it is truly quiet.